### PR TITLE
Moved projects to other folders

### PR DESCRIPTION
**Pull request by codingben**
Everything is tested and works fine.

### CODE REVIEW

**Comment by codingben** [comment]
@HughPH Hey

I also want to make cleaner code in the Box2D.NetStandard project, as I did yesterday. This will be another PR.

And things like this: 

https://github.com/benzuk/box2d-netstandard/blob/master/src/box2dx/Box2D.NetStandard/Dynamics/World/World.cs#L1084

Will be fixed. The break should be inside the brackets.

File: examples/console/ConsoleApp/Program.cs [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
 
             var ballDef = new BodyDef();
-            ballDef.Position = new Vector2(0, 0); Body ball = w.CreateBody(ballDef);
+            ballDef.Position = new Vec2(0, 0);
+
+            var ball = world.CreateBody(ballDef);
 
             var ballFixtureDef = new CircleDef();
-            ballFixtureDef.LocalPosition = new Vector2(0, 0);
+            ballFixtureDef.LocalPosition = new Vec2(0, 0);
```
Comment (Kira-PH) [inline examples/console/ConsoleApp/Program.cs]: Vec2 is deprecated. You should get warnings requesting that it's upgraded to System.Numerics.Vector2.

**Review by Kira-PH — COMMENTED** [review]
Why did you want to use Vec2 in the ConsoleApp? (We can remove that app, I was using it and a C++ equivalent to debug step through both codebases at once.)

File: examples/console/ConsoleApp/Program.cs (hunk 1 above)
Comment (codingben) [reply examples/console/ConsoleApp/Program.cs]: ConsoleApp uses the old code (version 1.0.3), but in this version there is no Vector2.

Comment (codingben) [reply examples/console/ConsoleApp/Program.cs]: https://github.com/benzuk/box2d-netstandard/blob/master/src/ConsoleApp1/Program.cs

Take a look at namespaces. It's master branch.

Comment (codingben) [reply examples/console/ConsoleApp/Program.cs]: With newer code, this should be Box2D.NetStandard.*. I was confused with Vector2, so I referenced the project to version 1.0.3 (nuget package).

**Comment by Kira-PH** [comment]
> And things like this:
> 
> https://github.com/benzuk/box2d-netstandard/blob/master/src/box2dx/Box2D.NetStandard/Dynamics/World/World.cs#L1084
> 
> Will be fixed. The break should be inside the brackets.

The break is outside the brackets in the original Box2D code! It's something I noticed and didn't get round to changing.

We have two ConsoleApps, both just drop a ball but one was to step through code. I think both are useless. I'm happy that the unit tests already cover that quite well.

**Review by Kira-PH — APPROVED** [review]

**Comment by codingben** [comment]
> > And things like this:
> > https://github.com/benzuk/box2d-netstandard/blob/master/src/box2dx/Box2D.NetStandard/Dynamics/World/World.cs#L1084
> > Will be fixed. The break should be inside the brackets.
> 
> The break is outside the brackets in the original Box2D code! It's something I noticed and didn't get round to changing.
> 
> We have two ConsoleApps, both just drop a ball but one was to step through code. I think both are useless. I'm happy that the unit tests already cover that quite well.

So it's okay to keep the break out of brackets? Let's agree - I won't make logical changes. I won’t touch anything except the code style (long lines, etc.), spaces, empty lines, etc. Does that sound good?

I'll remove ConsoleApp now.

**Comment by codingben** [comment]
@HughPH BTW - FYI: If you need, I created a small library for use in my server-side project:

https://github.com/benzuk/maple-fighters-server/tree/develop/lib/physics/box2d (Available as NuGet Package)

Maybe I will add this library for use in this repository. :)

**Comment by Kira-PH** [comment]
@benzuk 

> So it's okay to keep the break out of brackets? Let's agree - I won't make logical changes. I won’t touch anything except the code style (long lines, etc.), spaces, empty lines, etc. Does that sound good?

`break` outside brackets will work. The case statement without any brackets will work, too.

Feel free to make whatever changes you want - just make sure the unit tests pass :) Ideally, check that what you want to change is covered by unit tests. If not, create a unit test which covers it, then make changes.

**Comment by codingben** [comment]
> @benzuk
> 
> > So it's okay to keep the break out of brackets? Let's agree - I won't make logical changes. I won’t touch anything except the code style (long lines, etc.), spaces, empty lines, etc. Does that sound good?
> 
> `break` outside brackets will work. The case statement without any brackets will work, too.
> 
> Feel free to make whatever changes you want - just make sure the unit tests pass :) Ideally, check that what you want to change is covered by unit tests. If not, create a unit test which covers it, then make changes.

Gotcha. Thanks man!

**Pull request merged**